core-v1/src/main/scala/engine/StandardEngine.scala

```diff
@@ -212,6 +213,8 @@ object StandardEngine {
       ExprResult.IsEqual(expr, finalState)
     }
 
+    override def visitGetOrElse[I, O : OP](expr: Expr.GetOrElse[I, O, OP]): PO <:< I => ExprResult[PO, I, O, OP] = ???
```

Comment:
I assume these definitions will come in a later PR.

Reply:
This is a bug I resolved in a later commit. I should cherry-pick and rebase that commit here. Thanks for catching this!